exporter/modules.go

```diff
@@ -0,0 +1,51 @@
+package exporter
+
+import (
+	"strings"
+
+	"github.com/gomodule/redigo/redis"
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+)
+
+func (e *Exporter) extractModulesMetrics(ch chan<- prometheus.Metric, c redis.Conn) {
+	info, err := redis.String(doRedisCmd(c, "INFO", "MODULES"))
+	if err != nil {
+		log.Errorf("extractSearchMetrics() err: %s", err)
+		return
+	}
+
+	lines := strings.Split(info, "\r\n")
+	for _, line := range lines {
+		log.Debugf("info: %s", line)
+
+		split := strings.Split(line, ":")
```

Comment:
can we add a quick check here that `len(split) == 2`?  I'm not sure the switch statement would catch that despite the `case` in line 39

Reply:
Added. Though I thing case would catch that.
With this approach switch/case is not needed, so I replaced it with if.

Reply:
👍